test/smoke/src/areas/positron/welcome/welcome.test.ts

```diff
@@ -0,0 +1,176 @@
+			const app = this.app as Application;
+			const OPEN_BUTTONS_LABELS = process.platform === 'darwin' ?
+				['Open...', 'New Folder...', 'New Folder from Git...']
+				: ['Open File...', 'Open Folder...', 'New Folder...', 'New Folder from Git...'];
+
+			await expect(app.workbench.positronWelcome.startTitle).toHaveText('Start');
+
+			await expect(app.workbench.positronWelcome.startButtons).toHaveText(['New Notebook', 'New File', 'New Console', 'New Project']);
+
+			await expect(app.workbench.positronWelcome.helpTitle).toHaveText('Help');
+
+			await expect(app.workbench.positronWelcome.helpLinks).toHaveText(['Positron Documentation', 'Positron Community', 'Report a bug']);
+
+			await expect(app.workbench.positronWelcome.openTitle).toHaveText('Open');
+
+			await expect(app.workbench.positronWelcome.openButtons).toHaveText(OPEN_BUTTONS_LABELS);
+
+			await app.workbench.quickaccess.runCommand('File: Clear Recently Opened...');
+
+			await expect(app.workbench.positronWelcome.recentTitle).toHaveText('Recent');
+
+			// 'open a folder' is a button so there is no character space because of its padding
+			await expect(app.workbench.positronWelcome.recentSection.locator('.empty-recent')).toHaveText('You have no recent folders,open a folderto start.');
+		});
+
+		it('Click on new project from the Welcome page', async function () {
+			const app = this.app as Application;
+
+			await app.workbench.positronWelcome.newProjectButton.click();
+			await app.workbench.positronPopups.popupCurrentlyOpen();
+
+			await app.workbench.positronPopups.waitForModalDialogBox();
+
+			// confirm New Project dialog box is open
+			await app.workbench.positronPopups.waitForModalDialogTitle('Create New Project');
+
+			await app.workbench.positronPopups.clickCancelOnModalDialogBox();
+		});
+
+		describe('Python', () => {
+			before(async function () {
+				await PositronPythonFixtures.SetupFixtures(this.app as Application);
+			});
+
+			it('Create a new Python file from the Welcome page', async function () {
+				const app = this.app as Application;
+
+				await app.workbench.positronWelcome.newFileButton.click();
+
+				await app.workbench.quickinput.selectQuickInputElementContaining('Python File');
+
+				await expect(app.workbench.editors.activeEditor.locator(app.workbench.editors.editorIcon)).toHaveClass(/python-lang-file-icon/);
+
+				await app.workbench.quickaccess.runCommand('View: Close Editor');
+			});
+
+			it('Create a new Python notebook from the Welcome page', async function () {
+				const app = this.app as Application;
+
+				await app.workbench.positronWelcome.newNotebookButton.click();
+
+				await app.workbench.positronPopups.clickOnModalDialogPopupOption('Python Notebook');
+
+				await expect(app.workbench.editors.activeEditor.locator(app.workbench.editors.editorIcon)).toHaveClass(/ipynb-ext-file-icon/);
+
+				const expectedInterpreterVersion = new RegExp(`Python ${process.env.POSITRON_PY_VER_SEL}`, 'i');
```

Comment:
Look out for issues with "Python" already being in POSITRON_PY_VER_SEL:
https://github.com/posit-dev/positron/blob/main/.github/workflows/positron-ci.yml#L91
https://github.com/posit-dev/positron/blob/main/.github/workflows/positron-full-test.yml#L122
I _think_ "Python" can be removed from the CI files' ENV VARs, but as is I assume this would fail because you'd be grepping on something like "Python Python 3.10.12"